hack/generate.sh

```diff
@@ -56,6 +56,10 @@ function remove_dashes_and_dots() {
     done
 }
 
+function init_models_submodule() {
+    cd models && go mod init github.com/openshift/assisted-service/models && go mod tidy
+}
```

Comment:
To make ``cd`` scope only for this function, use:
```suggestion
function init_models_submodule() (
    cd models && go mod init github.com/openshift/assisted-service/models && go mod tidy
)
```

Or alternatively wrap ``cd models && ...`` with parenthesis

Reply:
I think that i will remove the automatic part. it may cause issues if we run generate from swagger in the CI and it will automatically update a module and complain about it. 
So it will be static and if the complication fails then the author of the PR will need to fix it, similar to what we have in the root directory, what do you think?

Reply:
that solution you did is actually a great idea. just removing ``*.go`` files